yt_dlp/extractor/tver.py

```diff
@@ -161,17 +176,212 @@ def _real_extract(self, url):
             ]
         ]
 
-        return {
-            '_type': 'url_transparent',
+        metadata = {
             'title': title,
             'series': series,
             'episode': episode,
             # an another title which is considered "full title" for some viewers
             'alt_title': join_nonempty(title, provider, onair_label, delim=' '),
             'channel': provider,
-            'description': str_or_none(video_info.get('description')),
             'thumbnails': thumbnails,
-            'url': smuggle_url(
-                self.BRIGHTCOVE_URL_TEMPLATE % (p_id, r_id), {'geo_countries': ['JP']}),
-            'ie_key': 'BrightcoveNew',
+            **traverse_obj(video_info, {
+                'description': ('description', {str}),
+                'timestamp': ('viewStatus', 'startAt', {int_or_none}),
+                'episode_number': ('no', {int_or_none}),
+            }),
+        }
+
+        if onair_label:
+            metadata.update(self._format_broadcast_date(onair_label))
+
+        if backend == 'brightcove':
+            p_id = video_info['video']['accountID']
+            r_id = traverse_obj(video_info, (
+                'video', ('videoRefID', 'videoID'), {str}, any, {require('reference ID')}))
+            if not r_id.isdecimal():
+                r_id = f'ref:{r_id}'
+
+            return {
+                **metadata,
+                '_type': 'url_transparent',
+                'url': smuggle_url(
+                    self.BRIGHTCOVE_URL_TEMPLATE % (p_id, r_id),
+                    {'geo_countries': ['JP']}),
+                'ie_key': 'BrightcoveNew',
+            }
+
+        ref_id = video_info['streaks']['videoRefID']
+        if not ref_id.startswith('ref:'):
+            ref_id = f'ref:{ref_id}'
+
+        return {
+            **self._extract_from_streaks_api(video_info['streaks']['projectID'], ref_id, {
+                'Origin': 'https://tver.jp',
+                'Referer': 'https://tver.jp/',
+            }),
+            **metadata,
+            'id': video_id,
+        }
```

Comment:
I removed the brightcove method and moved the code back into `_real_extract` because A) it wasn't that much code and B) here we also want to return `'_old_archive_ids'` with the brightcove id if possible